wgpu/src/backend/web.rs

```diff
@@ -1078,24 +1086,29 @@ impl crate::Context for Context {
 
         let possible_features = [
             //TODO: update the name
-            (wgt::Features::DEPTH_CLIP_CONTROL, Gfn::DepthClamping),
-            (
-                wgt::Features::DEPTH24UNORM_STENCIL8,
-                Gfn::Depth24unormStencil8,
```

Comment:
What happened to this?

Reply:
`DepthClamping` renamed `DepthClipControl`, `Depth24unormStencil8` removed from latest WebGPU spec